book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'person'
+
+class Book
+  attr_accessor :title, :person
+
+  def initialize(title, person)
+    @title = title
+    person.books << self
```

Comment:
- [ ] I would suggest you to initialize the person variable for the Book object before creating a bidirectional association between the Book and Person classes. Example snippet:

```ruby
def initialize(title, person)
    @title = title
    @person = person
    person.books << self 
  end
```